cmake_ext.py

```diff
@@ -147,8 +179,9 @@ def build_extension(self, ext):
         target = ext.name.split(".")[-1]
         suffix = sc.get_config_var("EXT_SUFFIX") or sc.get_config_var("SO")
         output_file = Path(extdir) / (target + suffix)
-        if not output_file.exists() or target == "_eposlhc":  # any target is fine
-            cmd = [cmake_exe, "--build", "."] + build_args
+        targets = [f"_{m}" for m in get_models()]
+        if not output_file.exists() or target == targets[0]:  # any target is fine
+            cmd = [cmake_exe, "--build", ".", "-t", *targets] + build_args
```

Comment:
There are 2 places in CMakeLists.txt, where there is condition for skip a target for Windows (IF WIN32...). I think that can be removed, if this chooses correct targets.

Reply:
I am not sure what you mean, but the target skipping for windows is now implemented in get_models() which defines the targets consistently everywhere.

Reply:
I meant to remove `if` conditions at line 286 and 361 of CMakeLists.txt.

Reply:
Ah, now I understand, yes, you are right.